components/member_list_team/member_list_team.tsx

```diff
@@ -134,7 +134,14 @@ export default class MemberListTeam extends React.Component<Props, State> {
         this.loadComplete();
     }
 
-    search = (term: string) => {
+    search = async (term: string) => {
+        await this.props.actions.getTeamMembers(this.props.currentTeamId, 0, this.props.users.length,
```

Comment:
When we search, we don't paginate so I set the limit to the number of users in the team.